src/main/java/br/com/blackbeard/blackbeardapi/exceptions/BarberArgumentException.java

```diff
@@ -0,0 +1,14 @@
+package br.com.blackbeard.blackbeardapi.exceptions;
+
+public class BarberArgumentException extends RuntimeException {
+
+    public static final String BARBER_NAME_CANNOT_BE_REPEATED = "barber name cannot be repeated";
```

Comment:
Mudar mensagem para "There is already a barber with that name for this barbershop"